pkgs/tools/misc/abduco/default.nix

```diff
@@ -1,29 +1,31 @@
-{ stdenv, fetchurl, writeText, conf? null}:
+{ stdenv, fetchFromGitHub, writeText, conf ? null }:
 
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
-    name = "abduco-0.6";
+  name = "abduco-2018-05-16";
 
-    meta = {
-        homepage = http://brain-dump.org/projects/abduco;
-        license = licenses.isc;
-        description = "Allows programs to be run independently from its controlling terminal";
-        maintainers = with maintainers; [ pSub ];
-        platforms = platforms.unix;
-    };
+  src = fetchFromGitHub {
+    owner = "martanne";
+    repo = "abduco";
+    rev = "8f80aa8044d7ecf0e43a0294a09007d056b20e4c";
+    sha256 = "0wqcif633nbgnznn46j0sng9l0wncppw1x1c42f75b4p9hrph203";
+  };
 
-    CFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-D_DARWIN_C_SOURCE";
+  configFile = optionalString (conf!=null) (writeText "config.def.h" conf);
+  preBuild = optionalString (conf!=null) "cp ${configFile} config.def.h";
 
-    src = fetchurl {
-        url = "http://www.brain-dump.org/projects/abduco/${name}.tar.gz";
-        sha256 = "1x1m58ckwsprljgmdy93mvgjyg9x3cqrzdf3mysp0mx97zhhj2f9";
-    };
+  CFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-D_DARWIN_C_SOURCE";
 
-    configFile = optionalString (conf!=null) (writeText "config.def.h" conf);
-    preBuild = optionalString (conf!=null) "cp ${configFile} config.def.h";
+  installPhase = ''
+    make PREFIX=$out install
```

Comment:
Please use `installFlags = [ "PREFIX=${placeholder "out"}" ];`

Reply:
I just left that as it was before, but it doesn't seem to be necessary at all

Reply:
I guess, when refactoring the layout, it is a good opportunity to clean up the code as well.